plane/source/docs/tuning-cruise.rst

```diff
@@ -1,37 +1,43 @@
+- Increase :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>`. (Increase :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` if an airspeed sensor or synthetic airspeed is not used)
 
-While the autopilot will operate correctly with minor offsets to the calibrated "level" attitude, optimum performance will occur if this adjusted to match that pitch attitude. This can be done by examining the average ``ATT.pitch`` value during level altitude cruise from dataflash logs, or visually if using an OSD or GCS, to adjust the horizon line level using slight adjustments to the :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` parameter value. This also will allow STABILIZE and FBWA modes to hold altitude at the same throttle level as used in the other modes.
+While the autopilot will operate correctly with minor offsets to the calibrated "level" attitude, optimum performance will occur if this is adjusted to match the pitch attitude that the autopilot tends to use in automatic throttle modes. This can be done by examining the average ``ATT.pitch`` value during level altitude flight from dataflash logs or in-flight on an artificial horizon (e.g., GCS or OSD HUD). Make slight adjustments to the :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` parameter value based on the value seen on the artificial horizon. Making these adjustments also will allow STABILIZE and FBWA modes to hold altitude at the same throttle level as used in the other modes.
 
-.. note:: It is important that the :ref:`STAB_PITCH_DOWN<STAB_PITCH_DOWN>` parameter be set correctly. This parameter will add "nose-down" trim when the throttle is lowered in pilot throttle controlled, stabilized modes, such as FBWA, to prevent the autopilot from holding the nose up as the plane slows down and potentially stalling. This can be tested at altitude in FBWA mode by moving the throttle to idle and checking that there is sufficient airspeed in a turn to avoid stalling (be prepared to do a stall recovery, which might occur). Increase the value of :ref:`STAB_PITCH_DOWN<STAB_PITCH_DOWN>`, if necessary.
+.. image:: ../../../images/AOA.jpg
 
-Adjusting CRUISE Mode Speed
-===========================
+The autopilot's priority is to obtain the correct combination of elevator and throttle to maintain constant altitude flight. How the autopilot does this is detailed in :ref:`tecs-total-energy-control-system-for-speed-height-tuning-guide`.
+
+The :ref:`STAB_PITCH_DOWN<STAB_PITCH_DOWN>` parameter will add "nose-down" trim when the throttle stick is lowered in pilot throttle controlled and stabilized modes, such as FBWA, to prevent the autopilot from holding the nose up as the plane slows down and potentially causing a stall. This can be tested, at altitude, in FBWA mode by moving the throttle to idle and checking that there is sufficient airspeed in a turn to avoid stalling. Be prepared to recover from a stall! Increase the value of :ref:`STAB_PITCH_DOWN<STAB_PITCH_DOWN>`, if necessary.
```

Comment:
this and probably should be moved to the acc cal page since it will be read by everyone (supposedly)...as a warning...its not really a cruise thing

Reply:
again this is what shows when you tag line 35 for a comment

Reply:
Moved to accel cal page.